pkg/util/workloads_utils.go

```diff
@@ -231,11 +232,13 @@ func GetEmptyWorkloadObject(gvk schema.GroupVersionKind) client.Object {
 		return nil
 	}
 
-	switch gvk.Kind {
-	case ControllerKindDep.Kind:
+	switch gvk.String() {
```

Comment:
What's wrong with the previous logic that compares GVK type?

Reply:
@FillZpp previous logic just consider `Kind`,  should we just consider `GroupKind`?

Reply:
So how about just compare the GVK type instead of converting to string?